### PR TITLE
TASK-41362: Improve snapshot attachments drawer

### PR DESCRIPTION
**Pull request by aelbenney**

### CODE REVIEW

File: apps/portlet-documents/src/main/webapp/vue-app/documents/components/Documents.vue [hunk 1]
```diff
@@ -45,7 +45,28 @@
...
+            document.openUrl = '';
+            document.breadCrumb = '';
+            document.id = '';
+            this.documents.unshift(document);
+            this.documents.pop();
+          });
+          localStorage.removeItem('newlyUploadedAttachments');
+          window.setTimeout(() => {
+            this.retrieveDocuments();
+          }, 5000);
```
Comment (boubaker) [inline apps/portlet-documents/src/main/webapp/vue-app/documents/components/Documents.vue]: Why waiting 5 seconds to retrieve newly uploaded documents ?

File: ecm-wcm-extension/src/main/webapp/attachments-app/components/AttachmentItem.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="attachment">
-    <v-list-item-avatar class="rounded-lg">
+    <v-list-item-avatar class="rounded-lg mr-3">
```
Comment (boubaker) [inline ecm-wcm-extension/src/main/webapp/attachments-app/components/AttachmentItem.vue]:
```suggestion
    <v-list-item-avatar class="rounded-lg me-3">
```

File: ecm-wcm-extension/src/main/webapp/attachments-selector/css/attachments.less [hunk 3]
```diff
@@ -1,1183 +1,1230 @@
 @import "variables.less";
 @import "mixins.less";
 
-.attachment {
```
Comment (boubaker) [inline ecm-wcm-extension/src/main/webapp/attachments-selector/css/attachments.less:4]: the modification of existing less files can impact old attachment components, thus I advice to define new CSS file for new component and avoir changing the ones used for old one, WDYT ?

Comment (aelbenney) [reply ecm-wcm-extension/src/main/webapp/attachments-selector/css/attachments.less:4]: I just added the old app parent class name to not affect our new app.

**Review by boubaker — DISMISSED** [review]

File: apps/portlet-documents/src/main/webapp/vue-app/documents/components/Documents.vue [hunk 4]
```diff
@@ -68,6 +76,11 @@
             documentsService.getRecentDocuments(this.limit).then(
               documents => {
                 this.documents = documents;
+                if (!documents.length) {
+                  localStorage.setItem('newlyUploadedAttachments', JSON.stringify({}));
```
Comment (boubaker) [inline apps/portlet-documents/src/main/webapp/vue-app/documents/components/Documents.vue:80]:
```suggestion
                  localStorage.setItem('newlyUploadedAttachments', '{}');
```

**Review by boubaker — APPROVED** [review]

**Pull request merged**